CONTRIBUTING.md

```diff
@@ -49,16 +49,20 @@ The Git LFS teams mark issues and pull requests with the following labels:
 * `roadmap` - A checklist issue with tasks to fulfill something from the
 [roadmap](./ROADMAP.md)
 
+## Branching strategy
+
+In general, contributors should develop on branches based off of `master` and pull requests should be to `master`.
+
```

Comment:
I would be happy to do it but I don't entirely understand your strategy for those.

I'm used to git-flow, myself, but if somebody would explain how you handle major and point releases (e.g. @technoweenie mentioned a script you use to squash fixes for point releases) I could take a stab at it.